src/helpers/Mail/templates/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
You can have a better folder structure: src/templates/mail/index.html

*Note* folder name should be lowercase

Reply:
This `src/templates/` will help us add other templates for different functionality.